examples/bert/run_glue_finetuning.py

```diff
@@ -215,15 +230,29 @@ def preprocess_data(inputs, labels):
         hidden_size=bert_config["hidden_size"],
         num_classes=3 if FLAGS.task_name in ("mnli", "ax") else 2,
     )
-    finetuning_model.compile(
-        optimizer=keras.optimizers.Adam(learning_rate=FLAGS.learning_rate),
-        loss="sparse_categorical_crossentropy",
-        metrics=["accuracy"],
-    )
-    finetuning_model.fit(
-        train_ds, epochs=FLAGS.epochs, validation_data=validation_ds
+
+    hypermodel = BertHyperModel(finetuning_model)
+
+    # Initialize the random search over the 4 learning rate parameters, for 4
+    # trials and 3 epochs for each trial.
+    tuner = kt.RandomSearch(
+        hypermodel=hypermodel,
+        objective="val_accuracy",
```

Comment:
Should we not just use val_loss here? Does the BERT paper mention what they do?

Reply:
I'm not sure. In the paper, they've mostly just highlighted the % increase in the accuracy but I might've missed something. I can try and experiment with `val_loss` if we want to go to that direction.

Reply:
@mattdangerw Should I just use `val_loss` or leave it as `val_accuracy` for now, because I didn't something written especially for the loss, just the accuracy.

Reply:
val_loss is fine for now! We can always change this later

Reply:
Yep, alright!